diff --git a/node_modules/chimp/lib/commands/generate.js b/node_modules/chimp/lib/commands/generate.js
index 120045a..16c7828 100644
--- a/node_modules/chimp/lib/commands/generate.js
+++ b/node_modules/chimp/lib/commands/generate.js
@@ -34,7 +34,7 @@ const fixGenerated = async (projectMainPath) => {
     await (0, execQuietly_1.execQuietly)(`node ${fixGeneratedPath}`, { cwd: projectMainPath });
 };
 const prettifyGenerated = async (projectMainPath, modulesPath = 'src') => {
-    await (0, execQuietly_1.execQuietly)(`npx prettier --write "${modulesPath}/**/*.ts" "generated/**/*.ts" --loglevel error`, {
+    await (0, execQuietly_1.execQuietly)(`npx prettier --write "${modulesPath}/**/*.ts" "generated/**/*.ts" --log-level error`, {
         cwd: projectMainPath,
     });
 };
diff --git a/node_modules/chimp/lib/generate/templates/schema.ts b/node_modules/chimp/lib/generate/templates/schema.ts
index c8cb587..93a9299 100644
--- a/node_modules/chimp/lib/generate/templates/schema.ts
+++ b/node_modules/chimp/lib/generate/templates/schema.ts
@@ -2,13 +2,13 @@
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
 // @ts-ignore
-import { buildFederatedSchema } from '@apollo/federation';
+import { buildSubgraphSchema } from '@apollo/subgraph';
 import { resolvers } from '{{generatedPrefix}}/graphql/resolvers';
 import gql from 'graphql-tag';
 
 const typeDefs = gql`{{{schemaString}}}`;
 
-const schema = buildFederatedSchema([
+const schema = buildSubgraphSchema([
   {
     typeDefs,
     resolvers,
